src/beacon-api/app/routes/firmware.py

```
@@ -0,0 +1,501 @@
+        firmware_id=firmware_uuid,
+        file_type=file_type,
+        bucket_name=settings.GCS_BUCKET_NAME
+    )
+    
+    # Parse range after we know file size
+    range_result = parse_range_header(range_header, file_size)
+    
+    if range_result:
+        range_start, range_end = range_result
+        # Download with range
+        file_data, file_size, blob_path, range_start, range_end = firmware_crud.download_firmware_file(
```

Ruff (0.14.7)

Unpacked variable `blob_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)